### PR TITLE
sayonara Gomunime, best anime site EVER..!!

### PR DESCRIPTION
**Pull request by hexated**
also added gomunimeis

### CODE REVIEW

File: Gomunimeis/src/main/kotlin/com/lagradost/Gomunimeis.kt [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+            TvType.TvSeries,
+        ) {
+            this.posterUrl = "$mainImageUrl/$image"
+            addSub(totalEpisode?.toIntOrNull())
+        }
+    }
+
+    override suspend fun search(query: String): List<SearchResponse> {
+        return app.get(
+            "$mainUrl/my-ajax?page=1&limit=10&action=load_search_movie&keyword=$query",
```
Comment (LagradOst) [inline Gomunimeis/src/main/kotlin/com/lagradost/Gomunimeis.kt:77]: because this is ajax you can override quicksearch to use search

**Review by LagradOst — APPROVED** [review]

File: Gomunimeis/src/main/kotlin/com/lagradost/Gomunimeis.kt (hunk 1 above)
Comment (hexated) [reply Gomunimeis/src/main/kotlin/com/lagradost/Gomunimeis.kt:77]: do you mean i have to duplicate this code like this ?
```kotlin
override suspend fun quickSearch(query: String): List<SearchResponse> {
        return app.get(
            "$mainUrl/my-ajax?page=1&limit=10&action=load_search_movie&keyword=$query",
            referer = "$mainUrl/search/?keyword=$query",
            headers = mapOf("X-Requested-With" to "XMLHttpRequest")
        ).parsedSafe<Responses>()?.data
            ?.mapNotNull { media ->
                media.toSearchResponse()
            } ?: throw ErrorLoadingException("Invalid Json reponse")
    }

    override suspend fun search(query: String): List<SearchResponse> {
        return app.get(
            "$mainUrl/my-ajax?page=1&limit=10&action=load_search_movie&keyword=$query",
            referer = "$mainUrl/search/?keyword=$query",
            headers = mapOf("X-Requested-With" to "XMLHttpRequest")
        ).parsedSafe<Responses>()?.data
            ?.mapNotNull { media ->
                media.toSearchResponse()
            } ?: throw ErrorLoadingException("Invalid Json reponse")
    }
```

Comment (LagradOst) [reply Gomunimeis/src/main/kotlin/com/lagradost/Gomunimeis.kt:77]: no, override suspend fun quickSearch(query: String): List<SearchResponse> = search(query)

**Comment by hexated** [comment]
ready to merge

**Pull request merged**